src/users/containers/user-details/user-details.component.ts

```diff
@@ -18,7 +19,9 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
   public user$: Observable<User>;
   public isLoading$: Observable<boolean>;
   public user: any;
+  public userAccessTypes: string[] = [];
 
+  private onDestory$ = new Subject<void>();
```

Comment:
Typo in "Destory".

Reply:
Fixed spelling for 'onDestroy' in commit:
https://github.com/hmcts/rpx-xui-manage-organisations/pull/1239/commits/58d56cf893eb33f411d8b82163bf30c343536915